src/main/java/io/confluent/connect/elasticsearch/RetryUtil.java

```diff
@@ -160,6 +160,7 @@ public static long computeRetryWaitTimeInMillis(int retryAttempts,
         );
         return function.call();
       } catch (IOException e) {
+        log.debug("Failed to {} due to {}. Retrying attempt ({}/{})", description, e.getCause(), attempt, maxAttempts);
```

Comment:
My 2 cents we should change this to warn.  The reason is that customer already told us that if he changes log to TRACE then it generates too much log, as this situation is now occurring once every few days we need to catch it in the act and that's why we should do it as warning.  The old PR also had this as warn https://github.com/confluentinc/kafka-connect-elasticsearch/pull/116/files#diff-b929a6768a74899db0afa5ad8eba4f2a0fafb65c9df9372085dfb01e840c02eeR361